src/assets/AccountTrackerController.ts

```diff
@@ -145,11 +146,17 @@ export class AccountTrackerController extends BaseController<
    * Refreshes all accounts in the current keychain.
    */
   refresh = async () => {
+    const { ethQuery } = this;
+
+    if (ethQuery === null) {
+      return;
```

Comment:
This looks like a change in behavior. It now returns nothing instead of throwing an error.

The same goes for `syncBalanceWithAddresses` - it now returns an empty object rather than throwing.